api/structure/schema.py

```diff
@@ -0,0 +1,48 @@
+import graphene
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+from api.structure.models import Structure as StructureModel
+from helpers.auth.authentication import Auth
+from helpers.auth.error_handler import SaveContextManager
+from utilities.validations import (
+    validate_empty_fields,
+)
+
+
+class Structure(SQLAlchemyObjectType):
+    """Autogenerated Return type for OfficeStructure"""
+
+    class Meta:
+        model = StructureModel
+
+
+class CreateOfficeStructure(graphene.Mutation):
+    """ Returns payload on structure creation"""
+    class Arguments:
+        web_id = graphene.String(required=True)
+        level = graphene.Int(required=True)
+        name = graphene.String(required=True)
+        parent_id = graphene.Int(required=True)
+        tag = graphene.String(required=True)
+        location_id = graphene.Int(required=True)
+        position = graphene.String(required=True)
+
+    structure = graphene.Field(Structure)
+
+    @Auth.user_roles('Admin')
```

Comment:
Hello @daniellamarr ,  we shall be receiving a list of dictionaries from the front-end(Kindly confirms this with Sinbad). we need to be able to save various dictionaries at the same time. If the thus is confirmed, Kindly look into being able to save several dictionaries at one. I will be available for assistance

Reply:
Oh okay
I will do that